README.md

```diff
@@ -73,18 +73,18 @@ This section is relevant only to internal users at SAS.
 
 ### Prerequisites
 
-* SAS Event Stream Processing running in Kubernetes with User Account and Authentication (UAA), Keycloak authentication or SAS Logon authentication.
-* A Grafana deployment with the name `grafana`, running in the same namespace as SAS Event Stream Processing.
-* It is recommended to have an Ingress for the Grafana deployment.
+* SAS Event Stream Processing running in the SAS Viya platform.
+* A Grafana deployment with the name `grafana`, this can be on a separate namespace to SAS Event Stream Processing.
```

Comment:
How about this:
A Grafana deployment with the name `grafana`. The Grafana deployment can be in the same namespace as SAS Event Stream Processing or in a separate namespace.